core/src/main/java/com/klaytn/caver/crpyto/KlayCredentials.java

```diff
@@ -66,15 +66,15 @@ public static KlayCredentials create(ECKeyPair ecKeyPair, String address) {
     public static KlayCredentials createWithKlaytnWalletKey(String klaytnWalletKey) {
         klaytnWalletKey = Numeric.cleanHexPrefix(klaytnWalletKey);
         String privateKey = klaytnWalletKey.substring(0, 64);
-        String address = klaytnWalletKey.substring(66);
+        String address = klaytnWalletKey.substring(68);
         address = Numeric.prependHexPrefix(address);
```

Comment:
Is this line necessary?

Reply:
For now, please ignore this comment and go ahead.
Please consider this later.

Reply:
No, I think it is not necessary. I would be better to remove the line. It will reflect your opinion on hotfix. Thanks!!